change/@itwin-tree-widget-react-606292c5-0d0f-4748-ad83-50279476f61b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Tree Widget: changed `CategoriesTreeComponentProps`, `ClassificationsTreeComponentProps` and `ModelsTreeComponentProps` to now accept optional viewport of `TreeWidgetViewport` type. It will be used instead of the `Viewport` from `@itwin/core-frontend`. Also, changed `TreeToolbarButtonProps`, `useCategoriesTreeButtonProps`, `UseCategoriesTreeProps`, `UseClassificationsTreeProps`, `useModelsTreeButtonProps` and `UseModelsTreeProps` to accept `TreeWidgetViewport` instead of `Viewport`. To migrate, use `createTreeWidgetViewport` and provide previously used `Viewport` from `@itwin/core-frontend`.",
```

- No need to prefix with "Tree widget" - we're in tree widget package.
- I'd rather mention components rather than their props.
- Don't think we need to exlicitly name every component that this change affected. Instead, let's focus on the change itself.

```suggestion
  "comment": "Changed props of `CategoriesTreeComponent`, `ClassificationsTreeComponent` and `ModelsTreeComponent` to accept an optional viewport of `TreeWidgetViewport` type, that would be used instead of the active AppUI's viewport. Also, changed props of other components that used to take `Viewport` to now take `TreeWidgetViewport` instead. Use the new `createTreeWidgetViewport` function to create `TreeWidgetViewport` from `Viewport`.",
```